test/conftest.py

```diff
@@ -16,7 +16,7 @@
 
 from . import load_fixture
 
-# pylint: disable=unused-argument
+# pylint: disable=protected-access, unused-argument
```

Comment:
We should not pylint our tests because it's a hell

Reply:
We don't do it on CI.